packages/calcite-components/src/components/menu-item/menu-item.scss

```diff
@@ -3,8 +3,12 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-menu-item-action-background-color: defines the background color of an action sub-component inside a menu-item.
+ * @prop --calcite-menu-item-action-indicator-color: defines the indicator color of an action sub-component inside a menu-item.
```

Comment:
A user can't enable an indicator on an Action rendered within Menu, I think we can remove this.

Reply:
should they be allowed to though? If not the indication in a menu-item might look different than an indicator elsewhere.

Reply:
I think we can leave it out, a user can't get to a state where the internally-rendered Action has an indicator. Only time the Action displays is when there is a populated `href` AND `breadcrumb is true: 

<img width="250" alt="Screenshot 2024-02-29 at 10 58 50 AM" src="https://github.com/Esri/calcite-design-system/assets/4733155/18ea18af-c660-4dee-9dd1-330c0c6a6028">

Or, for top level items, if the item has populated `href` AND slotted submenu items:
<img width="340" alt="Screenshot 2024-02-29 at 11 00 31 AM" src="https://github.com/Esri/calcite-design-system/assets/4733155/9c5709cc-1245-4989-a6c3-bbedf079e13e">